notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -1,1771 +1,1508 @@
 {
```

Comment:
<p>Reset section title to "Generate explanation metadata"</p><p> </p><p>Change the "We can use..." sentence to:</p><p> </p><p>"You can use the [Explainable AI SDK](https://pypi.org/project/explainable-ai-sdk/1.0.5/)</p><p> </p><p>Can you also add a line to print the metadata that's created, before exporting it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
The metadata contains the baseline so it's huge.